examples/simple/main.go

```diff
@@ -29,9 +29,8 @@ import (
 
 func main() {
 	svc, err := service.WithModules(
-		uhttp.New(registerHTTPers, []uhttp.Filter{simpleFilter{}}),
+		uhttp.New(registerHTTPers),
```

Comment:
Sorta, I meant to include it to get the build passing but now I merged your real fix and rebased.